hlx_statics/blocks/teaser/teaser.js

```diff
@@ -1,47 +1,50 @@
-import { decorateButtons, removeEmptyPTags, applyWidthOverride, applyBkgColorOverride, applyAnalyticHeaderOverride } from '../../scripts/lib-adobeio.js';
+import { decorateButtons, removeEmptyPTags } from '../../scripts/lib-adobeio.js';
 
-/**
- * Rearranges the links into a teaser-button-container div
- * @param {*} block The teaser block element
+/** 
+ * @param {Element} block
  */
 function rearrangeLinks(block) {
   const leftDiv = block.firstElementChild.firstElementChild;
   const teaserblockButton = document.createElement('div');
   teaserblockButton.classList.add('teaser-button-container');
-  leftDiv.querySelectorAll('p.button-container').forEach((p) => {
+
+  const buttons = leftDiv.querySelectorAll('p.button-container');
+  buttons.forEach((p) => {
     teaserblockButton.append(p);
   });
   leftDiv.append(teaserblockButton);
 }
 
-/**
- * Sets the background image and hides the image wrapper div
- * @param {Element} block The teaser block element
+/** 
+ * @param {Element} block
  */
 function setBackgroundImage(block) {
-  const imageWrapper = block.querySelector('picture');
-  const img = imageWrapper ? imageWrapper.querySelector('img') : null;
+  const img = block.querySelector('picture img');
 
   if (img) {
-    const src = img.src;
     const teaserContainer = block.closest('.teaser-container');
-    teaserContainer.style.backgroundImage = `url('${src}')`;
-    teaserContainer.style.backgroundSize = 'cover';
-    teaserContainer.style.backgroundPosition = 'center';
+    const imgParent = img.closest('picture').parentElement;
+    Object.assign(teaserContainer.style, {

```

Comment:
Nice, this is an immutable version of e.g. 
> teaserContainer.style.backgroundImage = `url('${img.src}')

Do we have a convention preference @timkim ? This is nice cus it's immutable, but it's not consistent with the other blocks so far.

It looks good to me, just curious how we deal with "code style" in this case